### PR TITLE
Fix #26: allow nested elements

### LINKED ISSUE #26: All elements are being removed by removeTags function
Function is [here](https://github.com/adobe-webplatform/balance-text/blob/cad7f2bbb961791f6add31639b17a63a0a71a375/jquery.balancetext.js#L46-L57), from the way it is written it seems like it should only remove brs and spans. but it is setting text as html.

[demo](http://jsbin.com/arewis/2/edit) *Sometimes you need to refresh the page to load the resources properly*

### PR DESCRIPTION
**Pull request by rileyjshaw**
Change the regexp in `isWS()` to only match space characters outside of element tags. The space in `<div class="foo">` is not matched, but the spaces in `<div>This is inner content</div>` are matched.

To make this more efficient, potential breakpoint indices could be held in an array outside of isWS.

A working demo can be seen [here](http://codepen.io/rileyjshaw/pen/oggBxz).

### CODE REVIEW

**Comment by redmunds** [comment]
@rileyjshaw Did you still want to finish this one?

**Comment by jpotter** [comment]
+1 on this fix; I've had to copy it into my files to enabled a tags in html, and it's solving the problem. (Thanks!)

**Comment by rileyjshaw** [comment]
@redmunds sorry that I haven't come back to this yet - if you think that you'll have time to patch it up before I can, go for it :)

File: jquery.balancetext.js [hunk 1]
```diff
@@ -99,10 +99,18 @@
     };
 
     /**
-     * Returns true iff c is an HTML space character.
+     * Returns true iff c is a space character outside of HTML < > tags.
      */
-    var isWS = function (c) {
-        return Boolean(c.match(/^\s$/));
+    var isWS = function (txt, index) {
+        var re = /\s(?![^<]*>)/g,
```
Comment (redmunds) [inline jquery.balancetext.js]: The `g` is needed. I think this increments the scan position.

**Comment by redmunds** [comment]
@rileyjshaw This code has been merged. Thanks for you contributions!
cc @jpotter

**Comment by rileyjshaw** [comment]
:+1: :+1: :+1:

**Pull request merged**